examples/kubernetes/cilium-metrics-svc.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cilium-metrics-svc
+  namespace: kube-system
+  labels:
+    k8s-app: cilium-metrics-svc
+    app: cilium-metrics-svc
+spec:
+  type: NodePort
+  ports:
+  - port: 9090
+    protocol: TCP
+    name: http-metrics
+  selector:
+    k8s-app: cilium
```

Comment:
Is this intended? My reading of this is that we're presenting a service for all cilium-agent pods, in order to collect the metrics. This shouldn't be necessary with if the ServiceMonitor generates a config similar to https://github.com/cilium/cilium/blob/master/examples/kubernetes/prometheus.yaml#L86-L117. 

If the intent was to expose the prometheus web ui we'd need something more like https://github.com/cilium/cilium/blob/master/examples/kubernetes/prometheus.yaml#L25-L27 (but I'm not sure I understand what was intended, so I'm probably wrong here).

Reply:
This service is only required when using prometheus-operator ServiceMonitor.

So, we need a service; then a ServiceMonitor that scrapes config from this service. 

I will update the README to make this clearer.